src/constants/colors.ts

```diff
@@ -5,7 +5,13 @@ export const COLORS = {
   BLOCK_TILE: "grey",
   EMPTY_TILE: "white",
   PATH_TILE: "yellow",
+  VISITED_LIST_TILE: "transparent",
   SCREEN_LAYOUT_BGC: "white",
+  BUILD_GRAPH_MENU_BORDER: "lightgray",
+  VISITED_TRANSITION_1: "#a902bf",
```

Comment:
I don't like the VISITED_TRANSITION_1
in COLORS
add
VISITED: {
    START: ....
    END: ....
}

access with COLORS.VISITED.START